src/main/java/lotto/domain/LottoTickets.java

```diff
@@ -28,4 +28,21 @@ public int size() {
     public List<Lotto> getLottos() {
         return new ArrayList<>(lottos);
     }
+
+    public Map<Rank, Integer> matchWith(WinningNumbers winningNumbers) {
```

Comment:
```suggestion
    public WinningResult matchWith(WinningNumbers winningNumbers) {
```
이미 구현해 놓은 WinningResult을 사용해 반환하면 어떨가?